### ISSUE TITLE
Carbon v10 Dropdown shows accessibility issues while running with DAP tool 

### ISSUE THREAD

**Issue by sakthi-nathiarasan** [carbon-design-system/carbon#5607]
## Environment

> Operating system
Linux- RHEL

> Browser
Chrome

> Automated testing tool and ruleset
IBM DAP tool

**Comment by sakthi-nathiarasan** [comment]
Hi,
     we are seeing the following accessibility issues with Dropdown while running DAP tool. 
The issues are,

1. Each form control must have associated label
2. Accessible name does not contain the visible text label

**Comment by sakthi-nathiarasan** [comment]
@joshblack  @amuqeetkhan

**Comment by sakthi-nathiarasan** [comment]
[
![image (3)](https://user-images.githubusercontent.com/62098142/76518145-9a92e400-6484-11ea-9465-085d666b9bd8.png)
![image (2)](https://user-images.githubusercontent.com/62098142/76518154-9f579800-6484-11ea-89e5-1e533c4ae671.png)

](url)

**Comment by emyarod** [comment]
from your screenshots there doesn't appear to be any label associated with your dropdown component

**Comment by sakthi-nathiarasan** [comment]
@emyarod , Here is our dropdown code,
                      <Dropdown
                      id="logDropdown"
                      className="logDropdown"
                      type="inline"
                      label="All Severities"
                      items={[
                        {
                          id: "all",
                          text: "All Severities",
                          label: "All Severities"
                        },
                        {
                          id: "error",
                          text: "Error",
                          label: "Error"
                        }
                      ]}
                      onChange={event => this.ss(event)}
                    />

Please let us know, why violation is coming or what we are missing here?

**Comment by emyarod** [comment]
@sakthi-nathiarasan can you create a [reduced test case in Code Sandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox) for easier testing and debugging?
